instrumentation/lettuce-4.3/src/main/java/com/lambdaworks/redis/RedisClient_instrumentation.java

```diff
@@ -0,0 +1,29 @@
+package com.lambdaworks.redis;
+
+import java.util.concurrent.TimeUnit;
+
+import com.lambdaworks.redis.api.StatefulRedisConnection;
+import com.lambdaworks.redis.codec.RedisCodec;
+import com.lambdaworks.redis.protocol.CommandHandler;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+
+@SuppressWarnings("deprecation")
+@Weave(originalName="com.lambdaworks.redis.RedisClient")
+public abstract class RedisClient_instrumentation extends AbstractRedisClient {
```

Comment:
Should be a capital `I` for instrumentation to keep it consistent with other modules: `RedisClient_Instrumentation`